kube-client/src/lib.rs

```diff
@@ -123,6 +123,44 @@ pub use crate::core::{CustomResourceExt, Resource, ResourceExt};
 /// Re-exports from kube_core
 pub use kube_core as core;
 
+pub mod prelude {
+    //! A "prelude" for kube client crate. Reduces the number of duplicated imports.
+    //!
+    //! This prelude is similar to the standard library's prelude in that you'll
+    //! almost always want to import its entire contents, but unlike the
+    //! standard library's prelude you'll have to do so manually:
+    //!
+    //! ```
+    //! use kube_client::prelude::*;
+    //! ```
+    //!
+    //! The prelude may grow over time as additional items see ubiquitous use.
+
+    #[allow(unreachable_pub)] pub use crate::client::ConfigExt as _;
+    pub use crate::{
+        api::{Ephemeral as _, Evict as _, Log as _},
+        core::prelude::*,
+    };
+}
```

Comment:
I don't think we need a prelude for each crate. One in the facade crate in `kube` is enough. We [discourage](https://kube.rs/crates/kube-client/) people to use the subcrates directly.

One prelude also removes the need for blanket imports from other crates (which is confusing for users to follow).